### PR TITLE
Improve readme: suggested Node version

### PR DESCRIPTION
**Pull request by jubeira**
# Description

Small doc update; selecting the correct Node version unequivocally is handy.

## Type of change

- [ ] Bug fix 
- [ ] New feature 
- [ ] Breaking change 
- [ ] Dependency changes
- [ ] Code refactor / cleanup
- [x] Documentation or wording changes
- [ ] Other

## Checklist:

- [x] The diff is legible and has no extraneous changes
- N/A - Complex code has been commented, including external interfaces
- N/A - Tests are included for all code paths
- [x] The base branch is either `master`, or there's a description of how to merge

## Issue Resolution

N/A

### CODE REVIEW

**Review by EndymionJkb — APPROVED** [review]
Should help! People might have different minor versions, so it's best not to give a specific version, like you have it.

File: README.md [hunk 1]
```diff
@@ -33,6 +33,12 @@ Active development occurs in this repository, which means some contracts in it m
 
 The build & test instructions below should work out of the box with Node ^14.18.0. (Please note that it needs Node 14 specifically, and will NOT work with Node 16 or higher. Minor version should be at least 18).
 
+To use the suggested Node version, run:
+
+```bash
+$ nvm use
+```
+
```
Comment (nventuro) [inline README.md:41]: This implies that nvm is used? I don't use nvm myself.

Comment (jubeira) [reply README.md:41]: yes; this is just a shortcut if you are lazy to install a version that works manually (the right versions are specified above these lines). Do you suggest rewording this?

Comment (EndymionJkb) [reply README.md:41]: I suppose it could be generalized a bit, noting that using an incompatible Node version can cause issues, and that it's possible to install multiple Node versions using a manager such as nvm. If you use nvm, you can switch between them with `nvm use`. I think that's the most popular one, so reasonable to mention it like this.
